modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/FDSView.tsx

```diff
@@ -94,9 +94,7 @@ const FDSView = ({
 }: IFDSViewProps) => {
 	const [activeIndex, setActiveIndex] = useState(0);
 	const [fdsView, setFDSView] = useState<FDSViewType>();
-	const [fieldTreeItems, setFieldTreeItems] = useState<Array<IFieldTreeItem>>(
-		[]
-	);
+	const [schemaFields, setSchemaFields] = useState<Array<IFieldTreeItem>>([]);
```

Comment:
The naming here is not ideal, as data structure contains tree-view specifics (`IFieldTreeItem`). If this variable held `Array<IField>`s, then it would make sense to call this something like `fields`, `schemaFields` or `apiSchemaFields`. Would it be better to forward it like `IField`s to children, and convert it to `IFieldTreeItem`s where needed? I'm not opposed to setting `IFieldTreeItem`s either, as eventually we will visualize those fields in tree-views everywhere. But in that case, I think we should keep calling this `fieldTreeItems`.

Reply:
So, if we are going to use tree-views everywhere I will change back to `fieldTreeItems`